dist/api_data.py

```diff
@@ -522,6 +522,13 @@ def __ge__(self, other):
             is not limited to not skewing newest, not favoring leaf pages,
             and modifying the eviction score mechanism.''',
             type='boolean'),
+        Config('log_retention', '0', r'''
+            adjust log archiving to retain at least this number of log files. (Warning: this
+            option can remove log files required for recovery if no checkpoints have yet been
+            done and the number of log files exceeds the configured value. As WiredTiger cannot
+            detect the difference between a system that has not yet checkpointed and one that
+            will never checkpoint, it might discard log files before any checkpoint is done.)''',
+            min='0', max='1024'),
```

Comment:
Does '0' mean keep all log files, or keep none?

Reply:
I think it should mean the value is simply ignored, so the default rules apply, whatever they may be. Documented in 2c05d4728.